pkg/api/index.go

```diff
@@ -38,8 +38,9 @@ func SearchIndexHandler(params index.SearchIndexParams) middleware.Responder {
 	var result []string
 	if params.Query.Hash != "" {
 		// This must be a valid sha256 hash
```

Comment:
SHA1 still seems to be supported?

Reply:
Good catch, I think that actually came later because we wanted to support git. So the comment is just out of date.